flowcraft/generator/inspect.py

```diff
@@ -1530,10 +1530,19 @@ def _get_run_hash(self):
 
         # Get name of the pipeline from the log file
         with open(self.log_file) as fh:
-            header = fh.readline()
-
-        # Regex supports absolute paths and relative paths
-        pipeline_path = re.match(".*\s([/\w/]*\w*.nf).*", header).group(1)
+            # Searches for the first occurence of the nextflow pipeline
+            # file name in the .nextflow.log file
+            while 1:
```

Comment:
This could be a function and you can call it in both files, instead of repeating the same code in the two files.

Reply:
You are right. It would be better to create a function to be used on both cases. Possibly adding a utils file for general functions.

Reply:
You can even define it in one of the files no matter which one and then import it on the other. But see if there isn't one file already with such type of general utility functions. Maybe @ODiogoSilva can help you here.